content/articles/how-does-a-dokku-work.md

```diff
@@ -0,0 +1,185 @@
+Title: How does a Heroku work?
+Date: 2015-01-12
+Short_summary: We explore how Heroku makes data from your Git repository appear as a web service.
```

Comment:
i would not mention heroku in the title, only in the intro.
Referring to dokku in the title is fine or talk about PaaS